### PR TITLE
add delete button

### PR DESCRIPTION
**Pull request by StephieTack**
right now the two more options function has still to be programmed

### CODE REVIEW

**Comment by F-Kirchhoff** [comment]
Hey Steffi,
I cannot give you a code review, if the feature is not finished. Do we want to take a look at it together?

**Comment by StephieTack** [comment]
@F-Kirchhoff now the issue is solved completely. could you have a look? :)

File: src/Components/ColorForm/ColorForm.jsx [hunk 1]
```diff
@@ -2,14 +2,14 @@ import ColorInput from "../ColorInput/ColorInput";
...
+  onAddColor,
   initialData = { role: "some color", hex: "#123456", contrastText: "#ffffff" },
 }) {
   function handleSubmit(event) {
     event.preventDefault();
     const formData = new FormData(event.target);
-    const data = Object.fromEntries(formData);
-    onSubmitColor(data);
+    const colorDa = Object.fromEntries(formData);
+    onAddColor(colorDa);
```
Comment (E-LeeSchaffer) [inline src/Components/ColorForm/ColorForm.jsx:12]: colorData instead of abbreviation?

Comment (E-LeeSchaffer) [inline src/Components/ColorForm/ColorForm.jsx:12]: shouldn't there be a definition of your colorDa object?
I would let line 11 with const data and then refer in colorDa to what you need from data...role, hex, contrastText

**Pull request merged**